### PR TITLE
Idealista/features/32

### PR DESCRIPTION
**Pull request by acuervof**
Allow set variable nginx_worker_rlimit_nofile
Added http_real_ip_module

### CODE REVIEW

**Review by jnogol — APPROVED** [review]

**Review by jmonterrubio — APPROVED** [review]

File: CHANGELOG.md [hunk 1]
```diff
@@ -4,6 +4,11 @@ This project adheres to [Semantic Versioning](http://semver.org/) and [Keep a ch
 
 ## [Unreleased](https://github.com/idealista/nginx-role/tree/develop)
 
+## [1.6.0](https://github.com/idealista/nginx-role/tree/1.6.0) (2017-10-26)
+- *[#32](https://github.com/idealista/nginx-role/issues/32) Support worker_rlimit_nofile setting* @acuervof
```
Comment (dortegau) [inline CHANGELOG.md]: This line should appear below "Full Changelog" 😇

File: defaults/main.yml [hunk 2]
```diff
@@ -17,6 +17,7 @@ nginx_service_state: started
 
 # Connection properties
 nginx_workers: auto
+#nginx_worker_rlimit_nofile --if you don't set FD's then OS settings will be used which is by default 2000
```
Comment (dortegau) [inline defaults/main.yml]: IMHO the example should declare a valid value

Comment (acuervof) [reply defaults/main.yml]: I had doubts about that when I was setting the variable. By default if the variable is not set, nginx get the value from OS settings http://nginx.org/en/docs/ngx_core_module.html#worker_rlimit_nofile

Finally I decided to replicate this behaviour with a conditional (https://github.com/idealista/nginx-role/pull/33/files#diff-754bfbe9e8e9c8a0ca748ca86030662e)

if you consider it necessary to assign a valid value I could set 2000 (OS default value)

File: CHANGELOG.md (hunk 1 above)
Comment (acuervof) [reply CHANGELOG.md]: You right, fixed!

File: defaults/main.yml (hunk 2 above)
Comment (dortegau) [reply defaults/main.yml]: Below I attach my proposal :)

```
# if you don't set FD's then OS settings will be used which is by default 2000
# nginx_worker_rlimit_nofile: 2000 
```

Comment (acuervof) [reply defaults/main.yml]: much more evident this way

**Review by dortegau — APPROVED** [review]

**Pull request merged**